<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dcomment</class>
 <widget class="QWidget" name="dcomment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(188, 203, 255);
	color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>评价菜品</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>亲，记得给好评哦~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bComDish5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>超级赞！（5分）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComDish4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>还不错！（4分）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComDish3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>一般般  （3分）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComDish2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>不够好  （2分）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComDish1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>真难吃！（1分）</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
